erpnext/docs/user/manual/en/setting-up/integrations/stripe-integration.md

```diff
@@ -18,6 +18,21 @@ Also it will create Payment Gateway Account entry. Payment Gateway Account is co
 
 After configuring Payment Gateway Account your system is able to accept online payments.
 
+### Setup subscriptions plans
+
+If you need to bill a recurring amount instead of a one-time charge, you can use Stripe's subscription system.
```

Comment:
We have shifted Docs to Foundation app. Please send a PR to foundation app. https://github.com/erpnext/foundation